Packet++/header/ArpLayer.h

```diff
@@ -93,7 +92,7 @@ namespace pcpp
 		 */
 		inline arphdr* getArpHeader() const
 		{
-			return (arphdr*)m_Data;
+			return reinterpret_cast<arphdr*>(m_Data);
```

Comment:
Should be fine to use `static_cast ` here?

Reply:
Tried, does not work. `static_cast` between unrelated pointers works only if one of the types is a `void*`.

Reply:
I see. Thanks for trying.